std/experimental/testing/attrs.d

```diff
@@ -0,0 +1,28 @@
+module std.experimental.testing.attrs;
```

Comment:
IMO, this module needs a little doc explaining that these structs are used with UDAs to attribute the unittests